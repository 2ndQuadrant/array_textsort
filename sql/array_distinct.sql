SELECT array_distinct('{"four","six","five","four","eleven"}');

SELECT array_distinct('{"four","six",NULL,"four","eleven"}');
