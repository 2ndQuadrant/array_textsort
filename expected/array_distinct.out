SELECT array_distinct('{"four","six","five","four","eleven"}');
     array_distinct     
------------------------
 {eleven,five,four,six}
(1 row)

SELECT array_distinct('{"four","six",NULL,"four","eleven"}');
ERROR:  array must not contain nulls
